torch/csrc/jit/frontend/script_type_parser.cpp

```diff
@@ -45,7 +45,7 @@ TypePtr ScriptTypeParser::subscriptToType(
       subscript_expr_types.push_back(parseTypeFromExprImpl(expr));
     }
     return TupleType::create(subscript_expr_types);
-  } else if (typeName == "List") {
+  } else if (typeName == "List" || typeName == "list") {
```

Comment:
Thanks for making the change! While we are at it, do you mind adding support for `Tuple` as well?